core/src/test/java/org/apache/spark/Java8RDDAPISuite.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package test.org.apache.spark.java8;
```

Comment:
for api tests it is still good to use test scope, because we wanted to make sure users can use this.